TaskList/java/src/main/java/com/codurance/training/tasks/TaskStatusService.java

```diff
@@ -0,0 +1,10 @@
+package com.codurance.training.tasks;
+
+import java.io.PrintWriter;
+import java.util.List;
+import java.util.Map;
+
+public interface TaskStatusService {
+
+    void setDone(Map<String, List<Task>> tasks,String idString, PrintWriter out);
+}
```

Comment:
If you are using TaskStatusService for multiple features than rename to have general thing instead for being specific to TaskStatusService..
For example implementing TaskStatusService for AddDeadline class doesn't make any sense